source/selection.html

```diff
@@ -1 +1,319 @@
-<!-- хоп хэй лалалэй  -->
+<!DOCTYPE html>
+<html class="page" lang="ru">
+  <head>
+    <meta charset="utf-8" />
+    <title>Каталог Cat Energy</title>
+  </head>
+  <body class="page__body">
+    <header class="page-header">
+      <a
+        class="page-header__logo-link"
+        aria-label="Перейти на главную страницу Cat Energy"
+        href="index.html"
+      >
+        <img
+          class="page-header__logo-image"
+          src="img/logo-mobile.svg"
+          width="34"
+          height="38"
+          alt="Логотип сайта Cat energy"
+        />
+        <img
+          class="page-header__logo-text"
+          src="img/logo-mobile-text.svg"
+          width="101"
+          height="18"
+          alt="Текстовый логотип сайта Cat energy"
+        />
+      </a>
+      <nav class="main-nav">
+        <button class="main-nav__toggler" type="button">
+          <span class="visually-hidden">Открыть меню</span>
+        </button>
+        <div class="main-nav__wrapper">
+          <ul class="main-nav__list menu">
+            <li class="menu__item">
+              <a class="menu__link" href="index.html">Главная</a>
+            </li>
+            <li class="menu__item">
+              <a class="menu__link" href="catalog.html">Каталог продукции</a>
+            </li>
+            <li class="menu__item menu__item--active">
+              <a class="menu__link" href="#">Подбор программы</a>
+            </li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+    <main class="page-main">
+      <section class="selection-form">
+        <div class="selection-form__wrapper">
+          <h1 class="selection-form__header">Подбор программы</h1>
+          <p class="selection-form__description">
+            Заполните анкету, и мы подберем<br />
+            программу питания для вашего кота
+          </p>
+          <form class="form" action="https://echo.htmlacademy.ru" method="POST">
+            <fieldset class="form__fieldset">
+              <legend class="form__legend visually-hidden">
+                Заполнить данные о питомце
+              </legend>
+              <div class="form__group">
```

Comment:
А это лучше + вынести это текстовое поле в отдельный бем блок, потому что с 99% вероятностью такой стиль будет распространяться на все виды таких текстовых контролов
```suggestion
              <div class="form__field text-field">
```